ext/civiimport/Civi/BAO/Import.php

```diff
@@ -203,7 +203,12 @@ public static function getFieldsForTable(string $tableName): array {
     $headers = UserJob::get(FALSE)
       ->addWhere('metadata', 'LIKE', '%' . $tableName . '%')
       ->addSelect('metadata')->execute()->first()['metadata']['DataSource']['column_headers'] ?? [];
-    $result = CRM_Core_DAO::executeQuery("SHOW COLUMNS FROM $tableName");
+    try {
+      $result = CRM_Core_DAO::executeQuery("SHOW COLUMNS FROM $tableName");
+    }
+    catch (\PEAR_Exception $e) {
+      throw new CRM_Core_Exception('Import table no longer exists');
+    }
```

Comment:
Is this part still necessary, if the `CRM_Core_Error::exceptionHandler()` now emits  `CRM_Core_Exception`?

Reply:
@totten - no it is part of one of the PRs to be rebased out once merged first

Reply:
(merged & up-merged to master)

Reply:
Ooh, I see. You're aiming for that variant to go into `5.59` - and then this variant goes into `master` (but it needs to be reconciled with whatever happened in `5.59`).

Reply:
I just rebased out this patch cos I'm trying to deal with another issue in the same file & it will conflict